src/gl/canvas2dRe.re

```diff
@@ -1,4 +1,6 @@
 type t;     /* Main type, representing the 2d canvas rendering context object */
+type gradientT;
```

Comment:
I'd prefer types to not have the `T` suffix. The `Gl` module is really the odd one out here as the only one that uses that convention.